code/modules/clothing/spacesuits/rig/rig.dm

```diff
@@ -433,7 +433,8 @@
 			malfunction()
 
 		for(var/obj/item/rig_module/module in processing_modules)
-			cell.use(module.Process() * CELLRATE)
+			if(!cell.use(module.Process() * CELLRATE))
```

Comment:
this seems a bit of a hack to begin with, don't expand upon it. 

Theres no reason that Process should always be associated with power costs, I find it hard to believe that's so in every case

Reply:
At least check whether a nonzero number is being returned

Reply:
Oh yes, my bad. Forgot to add check if cost is 0. This was made to disable modules once we out of power

Reply:
Oh yes, my bad. Forgot to add check if cost is 0. This was made to disable modules once we out of power